lib/client/node.js

```diff
@@ -311,6 +311,15 @@ class NodeClient extends Client {
 
     return this.call('rescan', start);
   }
+
+/**
+  * Abort scanning blockchain
+  * @returns {Promise}
+*/
+
+  abortrescan() {
```

Comment:
```suggestion
  abortRescan() {
```

big bug! so this I only discovered because I ran the tests through remote wallet as well as wallet plugin!